metrics/web/test/test_utils/auth_notifier_stub.dart

```diff
@@ -18,11 +19,12 @@ class AuthNotifierStub extends ChangeNotifier implements AuthNotifier {
   @override
   String get passwordErrorMessage => null;
 
-  /// Contains a user's authentication status.
-  bool _isLoggedIn;
+  /// Contains a state of the user authorization.
```

Comment:
🏷️ 
```suggestion
  /// Contains a user's authorization state.
```
Please, apply this suggestion to all similar situations.